src/app/pages/wishlist/component/wishlist/wishlist.component.html

```diff
@@ -0,0 +1,18 @@
+<div
+  *ngIf="favourites.length; else emptyList"
+  class="flex flex-col md:flex-row items-center justify-center w-full flex-wrap order-1 md:order-0 pb-48 md:pb-36 mt-16 mb-6">
```

Comment:
can you show me where we use the `mt-16 mb-6` classes please?

Reply:
I removed mb-6, it is not necessary, but the UI looks better with mt-16, because of the distance between the blue menu and the gray component header. But it is removable, of course :)